Resources/translations/SonataAdminBundle.en.xliff

```diff
@@ -202,6 +202,14 @@
                 <source>btn_delete</source>
                 <target>Yes, delete</target>
             </trans-unit>
+            <trans-unit id="label_list_batch_select">
+                <source>label_list_batch_select</source>
+                <target>Select / deselect all rows</target>
```

Comment:
german translation:
`Alle Zeilen anwählen / abwählen`

@core23 what would you say?